screwdriver.yaml

```diff
@@ -0,0 +1,21 @@
+---
+# Job definition block
+# "main" is a default job that all pipelines have
+jobs:
```

Comment:
We could add that to this one. I just imagined that the quick start configuration is super minimal. 

Thinking about it from a different angle, it could be better by providing one that has the basics covered.